configure.ac

```diff
@@ -538,26 +538,47 @@ fi
 
 dnl libcurl
 
-AC_ARG_WITH([libcurl],
-    [AS_HELP_STRING([--with-libcurl[[=PATH]]], [Specify libcurl path])], [], [with_libcurl=check])
-
-if test "x$with_libcurl" != xno
-then
-  CF3_WITH_LIBRARY(libcurl, [
-    AC_CHECK_LIB(curl, curl_global_init,
-      [],
-      [  libcurl_lib_found=no;
-         if test "x$with_libcurl" != xcheck;
-         then
-             AC_MSG_ERROR(Cannot find libcurl library);
-         fi
-      ])
-    AC_CHECK_HEADERS([curl/curl.h],
-      [libcurl_header_found=yes],
-      [if test "x$with_libcurl" != xcheck; then AC_MSG_ERROR(Cannot find libcurl header files); fi])
-  ])
+if [[[ $( which -s pkg-config ) -eq 0 ]]]; then
+   echo "db: pkg-config found";
```

Comment:
Please drop these debug messages.

Reply:
Done in p.r. 3500. For some reason, the pull triggered a new number. Please merge.